gravitational/next
# Update Product and Tech Paper og:image

- adds `previewImage` to yaml, which is passed in as the `image` prop in the [`Head` component](https://github.com/gravitational/next/blob/main/components/Head/Head.tsx) in order to render unique og:images.

**To verify:**
Check og:image image field in `<head />` of each page. 

Previously this value was `https://goteleport.com/static/og-image.png` as shown:
<img width="1190" alt="Screen Shot 2022-03-18 at 12 44 51 PM" src="https://user-images.githubusercontent.com/70108137/159046347-f8125abc-ec0f-45df-bfaf-3a071fdb2176.png">

Og:image should now be unique. For example:
<img width="1240" alt="Screen Shot 2022-03-18 at 12 42 42 PM" src="https://user-images.githubusercontent.com/70108137/159046474-f08177fa-620d-4627-bfe5-bbf67f62a742.png">

### Preview pages
**Product:**
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/ssh-server-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/kubernetes-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/application-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/database-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/desktop-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/features/

**Tech Papers:**
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/2021-infrastructure-access-report/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/top-10-hacks-of-the-past-decade/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/fedramp-how-to/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/practical-guide-to-secure-ssh-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/security-best-practices-for-pam/ 
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/access-and-security-trade-offs/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/secure-infrastructure-access/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/teleport-vs-aws-session-manager/
https://teleport-docs-next-git-delia-update-og-images-gravitational.vercel.app/resources/white-papers/zero-trust-access-management-best-practices/

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201974496096809
